docs/graphql/manual/actions/async-actions.rst

```diff
@@ -0,0 +1,35 @@
+Async Actions
+=============
+
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+WORK IN PROGRESS
+
+Sometimes you may not want to wait for an action to complete (say if the business logic takes a long time). In such cases you can create an **asynchronous** action, which returns an ``action_id`` immediately to the client before contacting the webhook.
```

Comment:
Love this feature! In trying this, I noticed that when running an async mutation, I am seeing `id` being made available through the GraphQL schema when attempting to subscribe to my action - as opposed to the `action_id` stated here.